lab-beeksma/test/fp.test.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const { expect } = require('chai');
+const arr = require('../lib/fp.js');
+
+const testArray = [1,2,3,4,5,6];
+const testArray2 = [7,8,9,10,11,12];
+const mapResults = [2,4,6,8,10,12];
+const filterResults = [2,4,6];
+const reduceResults = 21;
+const concatResults = [1,2,3,4,5,6,7,8,9,10,11,12];
+const spliceDeleted = [3,4];
+const spliceDone = [1,2,'z','y',5,6,7,8,9,0];
+
+
+describe('Functions with Call', function(){
+  describe('#nMap', function(){
+    it('should return new array',function(){
+      let result = arr.nMap(testArray,function(x){return x*2;});
+      console.log(result);
+      expect(result).to.be.an('array');
+      expect(result).to.be.deep.equal(mapResults);
```

Comment:
While it's common to extract a variable for reused test input (e.g. a `testArray` used in each test), tests are almost always easier to work with if the expected values are placed inline here rather than in variables above.